content/en/docs/ambient/architecture/index.md

```diff
@@ -9,57 +9,109 @@ owner: istio/wg-networking-maintainers
+the `productpage-viewer` policy to the `productpage` service in the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: productpage-viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Service
+        group: ""
+        name: productpage
+    {{< /text >}}
+
 ## Traffic routing
 
 In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
-1. **Uncaptured:** this is a standard pod without any mesh features enabled.
-1. **Captured:** this is a pod that has traffic intercepted by {{< gloss >}}ztunnel{{< /gloss >}}. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
-1. **Waypoint enabled:** this is a pod that is "Captured" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed.
-  A waypoint will, by default, apply to all pods in the same namespace.
-  It can optionally be set to apply to only a specific service account with the `istio.io/for-service-account` annotation on the `Gateway`.
-  If there is both a namespace waypoint and service account waypoint, the service account waypoint takes precedence.
+1. **Out of Mesh:** this is a standard pod without any mesh features enabled.
+1. **In Mesh:** this is a pod that has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policy can be enforced for pod traffic. This mode can be enabled for a pod by setting the `istio.io/dataplane-mode=ambient` label on the pod's namespace. This will enable *in mesh* mode for all pods in that namespace.
+1. **Waypoint enabled:** this is a pod that is "In Mesh" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed. To enforce L7 policies, add the `istio.io/use-waypoint` label to your resource to use waypoint for the labeled resource.
+  - If a namespace is labeled with `istio.io/use-waypoint` with its default waypoint for the namespace, the waypoint will apply to all pods in the namespace.
+  - The `istio.io/use-waypoint` label can also be set on individual services or pods when using a waypoint for the entire namespace is not desired.
+  - If the `istio.io/use-waypoint` label exists on both a namespace and a service, the service waypoint takes
+  precedence over the namespace waypoint as long as the service waypoint can handle service or all traffic.
+  Similarly, a label on a pod will take precedence over a namespace label.
 
 Depending on which category a workload is in, the request path will be different.
 
 ### Ztunnel routing
 
 #### Outbound
 
-When a captured pod makes an outbound request, it will be transparently redirected to ztunnel which will determine where and how to forward the request.
+When a pod in ambient mesh makes an outbound request, it will be transparently redirected to ztunnel which will determine where and how to forward the request.
```

Comment:
I'd say either a `pod in ambient mode` or `pod in an ambient mesh`. I just picked one
```suggestion
When a pod in an ambient mesh makes an outbound request, it will be transparently redirected to ztunnel which will determine where and how to forward the request.
```